docs/user/shared/admonition-rtd-business.rst

```diff
@@ -0,0 +1,2 @@
+.. note::
```

Comment:
I think we probably need to exclude this or mark it as an orphan to not Sphinx complain. Alternatively we could use a `txt` ending for it.

Reply:
:+1: on excluding https://github.com/readthedocs/readthedocs.org/blob/52d47666fc4c27aed903ad29d6a874986602fd72/docs/conf.py#L74-L74, changing the ext we lose the nice syntax hl in gh/editors.

Reply:
Oh thanks for noticing this! I was expecting some warning about not being included in TOC, so I didn't investigate further. Adding it to `exclude_patterns` seems like the nice bolstered version :+1:

Reply:
I've really tried everything in `exclude_patterns` and somehow on every build (even after `make clean`), I keep getting `_build/user/html/shared/admonition-rtd-business.html` in the outputs but with no incoming links or warnings.

It doesn't match what I would expect from https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-exclude_patterns

Reply:
I tried both with and without `user/` in front.

I can try with `{docset}` :+1:

Reply:
ahhhh, I noticed that override some time ago, but forgot to send a PR removing it :(

Reply:
I had a feeling that our documentation re-builds were oddly slow, and I think this brings them back to "normal".

(changing a single .rst file normally generates the new documentation in a second, but it was taking a lot of seconds)

Reply:
Also, linting could have caught this. Should investigate...

Reply:
I've tried to re-add the shadowing redefinition of `exclude_patterns` in `conf.py` in order to test if there is a linter for pre-commit that can catch this -- sadly neither autoflake, flake8 or pylint can catch this.

There might be a reason for it.

But now I tried and I'm afraid that we will have to still bother with these things in the future :)

I did really like pylint, though, it's very thorough.